collector/runtime-benchmarks/bufreader/src/main.rs

```diff
@@ -0,0 +1,30 @@
+use benchlib::benchmark::BenchmarkSuite;
+use snap::{read::FrameDecoder, write::FrameEncoder};
+use std::io::{BufRead, BufReader, Write};
+
+const BYTES: usize = 64 * 1024 * 1024;
+
+fn main() {
+    let mut suite = BenchmarkSuite::new();
+
+    suite.register("bufreader-snappy", || {
```

Comment:
Could you please include a short comment above this benchmark that describes what does it measure, possibly with a link to the regression issue?